src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/CachedCertificateValue.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace System.Net.Http
+{
+    internal sealed class CachedCertificateValue(byte[] rawCertificateData, long lastUsedTime)
+    {
+        public byte[] RawCertificateData { get; } = rawCertificateData;
+        public long LastUsedTime { get; set; } = lastUsedTime;
```

Comment:
Not sure how much we care, but since this is being used by multiple threads it could result in tearing when running as 32-bit.

A possible fix could be
```c#
internal sealed class CachedCertificateValue(byte[] rawCertificateData, long lastUsedTime)
{
    private long _lastUsedTime = lastUsedTime;
    
    public byte[] RawCertificateData { get; } = rawCertificateData;

    public long LastUsedTime
    {
        get => Volatile.Read(ref _lastUsedTime);
        set => Volatile.Write(ref lastUsedTime, value);
    }
}
```